lxd/device/nic_routed.go

```diff
@@ -1,12 +1,13 @@
 package device
 
 import (
+	"context"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
Import ordering is incorrect here. It was correct before so I guess some automated tool on your system messed it up :)

Reply:
I've set `gofmt` as the file watcher so I'm not sure what is doing this!